backend/de.metas.manufacturing/src/main/sql/postgresql/ddl/views/PP_Maturing_Candidate_v.sql

```diff
@@ -0,0 +1,40 @@
+DROP VIEW IF EXISTS PP_Maturing_Candidates_v
+;
+
+CREATE OR REPLACE VIEW PP_Maturing_Candidates_v AS
+SELECT hu.m_hu_id                                                 AS PP_Maturing_Candidates_v_ID,
+       hu.m_hu_id,
+       hus.qty,
+       hu.hustatus,
+       pp.m_product_id,
+       hus.c_uom_id,
+       w.m_warehouse_id,
+       mc.m_maturing_configuration_id,
+       mcl.m_maturing_configuration_line_id,
+       pp.pp_product_planning_id,
+       pp.m_attributesetinstance_id,
+       hua.valuedate + MAKE_INTERVAL(0, mcl.maturityage::integer) AS DateStartSchedule,
+       ppoc.pp_order_candidate_id,
+       hu.ad_org_id,
+       hu.ad_client_id
+FROM m_hu hu
+         INNER JOIN m_hu_storage hus ON hu.m_hu_id = hus.m_hu_id
+         INNER JOIN m_hu_attribute hua ON hu.m_hu_id = hua.m_hu_id AND m_attribute_id = (SELECT m_attribute_id FROM m_attribute WHERE VALUE = 'ProductionDate')
+         INNER JOIN m_locator l ON hu.m_locator_id = l.m_locator_id
+         INNER JOIN m_warehouse w ON l.m_warehouse_id = w.m_warehouse_id
+         INNER JOIN m_maturing_configuration_line mcl ON hus.m_product_id = mcl.from_product_id AND mcl.isactive = 'Y'
+         INNER JOIN m_maturing_configuration mc ON mcl.m_maturing_configuration_id = mc.m_maturing_configuration_id AND mc.isactive = 'Y'
+         INNER JOIN pp_product_planning pp ON w.m_warehouse_id = pp.m_warehouse_id AND mcl.m_maturing_configuration_line_id = pp.m_maturing_configuration_line_id AND pp.m_product_id = mcl.matured_product_id AND pp.isactive = 'Y' AND pp.ismatured = 'Y'
```

Comment:
do we have some unique index or something that makes sure this INNER JOIN won't return more than one PP_Product_Planning?

Reply:
added
```SQL
CREATE UNIQUE INDEX pp_product_planning_maturing_uc
    ON pp_product_planning (ad_org_id, m_warehouse_id, m_maturing_configuration_line_id, m_product_id)
    WHERE (isactive = 'Y'::bpchar AND ismatured = 'Y')
;
```
There is no need for data migration because it's only for matured product plannings.